test/dotnet-run.UnitTests/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-*"
```

Comment:
The netcore app dependencies are a special case because we generate in the same build. 

this is part of the impetus to split the builds apart.